vcbuild.bat

```diff
@@ -85,19 +80,21 @@ if defined build_release (
   set config=Release
   set msi=1
   set licensertf=1
-  set download_arg="--download=all"
+  set config_flags=%config_flags% --download=all
   set i18n_arg=small-icu
 )
 
-if "%config%"=="Debug" set debug_arg=--debug
-if defined nosnapshot set snapshot_arg=--without-snapshot
-if defined noetw set noetw_arg=--without-etw& set noetw_msi_arg=/p:NoETW=1
-if defined noperfctr set noperfctr_arg=--without-perfctr& set noperfctr_msi_arg=/p:NoPerfCtr=1
+if "%config%"=="Debug" set config_flags=%config_flags% --debug
+if defined nosnapshot set config_flags=%config_flags% --without-snapshot
+if defined noetw set config_flags=%config_flags% --without-etw& set noetw_msi_arg=/p:NoETW=1
+if defined noperfctr set config_flags=%config_flags% --without-perfctr& set noperfctr_msi_arg=/p:NoPerfCtr=1
 if defined RELEASE_URLBASE set release_urlbase_arg=--release-urlbase=%RELEASE_URLBASE%
 
-if "%i18n_arg%"=="full-icu" set i18n_arg=--with-intl=full-icu
-if "%i18n_arg%"=="small-icu" set i18n_arg=--with-intl=small-icu
-if "%i18n_arg%"=="intl-none" set i18n_arg=--with-intl=none
+if "%i18n_arg%"=="full-icu" set config_flags=%config_flags% --with-intl=full-icu
+if "%i18n_arg%"=="small-icu" set config_flags=%config_flags% --with-intl=small-icu
+if "%i18n_arg%"=="intl-none" set config_flags=%config_flags% --with-intl=none
+
+if defined CONFIG_FLAGS set config_flags=%config_flags% %CONFIG_FLAGS%
```

Comment:
..so if you want to achieve passing additional config flags through an environment var, you would have to name it something different. E.g.: additional_config_flags, or name the other one differently e.g. configure_flags, or _configure_flags